### DISCUSSION TITLE
How to delete relations between to table when used link to another record

### DISCUSSION THREAD

**Discussion by qjcjerry** [nocodb/nocodb#490] [Q&A]
When user LinktoAnotherRecord as a column and adding data, you can not delete the column , because of error.
And the child record  can not delete either.
![image](https://user-images.githubusercontent.com/6712585/130394428-d5e37608-43fc-446c-a3a5-8062e135a110.png)
 
![image](https://user-images.githubusercontent.com/6712585/130394511-79891d6a-9845-4945-ae25-efa6e192fb40.png)

**Comment by o1lab** [comment]
Fix for this is coming today. This is since we changed knex to rely on custom foreignKey name. This was default before, however some of our users faced key name being too long.
